scripts/new-customer-key.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+function usage {
+    echo "Usage: $0 [sw|fw] customer_name version"
+    echo -e "\t [sw|fw] - either fw or sw is mandatory, parameter place 'open-source software' or 'open-source firmware' in gpg key real name."
+    echo -e "\t customer_name - customer name in quotes e.g. \"PC Engines\", if customer_name directory does not exist it would be created"
+    echo -e "\t version - version e.g. \"4.16\", \"1.x\", if version exist error would be returned"
+    exit 1
+}
+
+
+[ $# -ne 3 ] && echo "ERROR: invalid number of parameters" && usage && exit 1
+[ -z "$2" ] && echo "ERROR: customer_name is null" && exit 1
+[ -z "$3" ] && echo "ERROR: version is null" && exit 1
+subcmd=$1
+customer_name=$2
+version=$3
+
+case $subcmd in
+    sw)
+	real_name="${customer_name} open-source software release ${version} signing key"
+	# 3mdeb Open Source Software Master Key <contact@3mdeb.com>
+	signing_kid=6058A306EF96BCBF56567B8C80693E028589B763
+	;;
+    fw)
+	real_name="${customer_name} open-source firmware release ${version} signing key"
+	# 3mdeb Dasharo Master Key
```

Comment:
PC Engines is an exception. The fw key is signed by https://github.com/3mdeb/3mdeb-secpack/blob/master/open-source-firmware/3mdeb-open-source-firmware-master-key.asc

Reply:
Resovled by https://github.com/3mdeb/3mdeb-secpack/pull/45